chromium_src/base/trace_event/builtin_categories.h

```diff
@@ -0,0 +1,21 @@
+/* Copyright (c) 2021 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_CHROMIUM_SRC_BASE_TRACE_EVENT_BUILTIN_CATEGORIES_H_
+#define BRAVE_CHROMIUM_SRC_BASE_TRACE_EVENT_BUILTIN_CATEGORIES_H_
+
+// Please add all brave categories here. For small features use the existing
+// 'brave' category. For new big subsystems with a lot of traces create an
+// separated category named 'brave.<feature_name>`.
```

Comment:
I would add two more points:
 * pls read the original `builtin_categories.h` for more details
 * pls keep the list sorted

Reply:
added